UglyTicTacToe/Models/Buttons.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+
+namespace UglyTicTacToe.Models
+{
+    public enum MyState
+    {
+        None,
+        Cross,
+        Nill
+    }
+
+    public class Buttons
+    {
+        public static MyState recentState { get; set; }
+        public MyState currentState;
+        public Buttons(MyState state)
+        {
+            this.currentState = state;
+        }
+    }
```

Comment:
In lieu of a class that has a single property, I'd just create an array of MyState enumeration values (i.e. `MyState[,]`). The static `recentState` property could be moved to your `GameLogic` class. Also, consider renaming the `MyState` enum to `ButtonState`?